inference-engine/thirdparty/clDNN/api/loop.hpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+#include <vector>
+#include <functional>
+#include "primitive.hpp"
+#include "topology.hpp"
+
+#define DEFAULT_MAX_NUM_ITERATION 256
```

Comment:
This is still not quite clear for me. Where this max iterations number comes from? Is it defined in ngraph spec? Why exactly 256? It was 128 in the previous version

Reply:
To be honest the value is not considered seriously. I think we'd better ask for a suggestion from MO team..?

Reply:
The reason why `DEFAULT_MAX_NUM_ITERATION` is changed from 128 to 256 is that some model's num_iteration is greater than 128 and I think the `DEFAULT_MAX_NUM_ITERATION` should be larger enough to cover most of models.